src/include/duckdb/common/types/time.hpp

```diff
@@ -37,6 +37,8 @@ class Time {
 	DUCKDB_API static string ToUTCOffset(int hour_offset, int minute_offset);
 
 	DUCKDB_API static dtime_t FromTime(int32_t hour, int32_t minute, int32_t second, int32_t microseconds = 0);
+	DUCKDB_API static bool TryFromTime(int32_t hour, int32_t minute, int32_t second, int32_t microseconds,
```

Comment:
I don't think this is needed if you use my cleanup.

Reply:
I think it's clean though in the sense that the function ends up in `time.hpp` which mirrors the structure of date and timestamp.

Reply:
I'll merge your PR though, I guess it effectively increases performance.

Reply:
I cherry-picked your commit, hope that works for you.

Reply:
I left these in though, for completeness one could say, time then has the same functions as date and timestamp. I can delete them though if you think that's more suitable.

Reply:
Yes please remove - dates and times are not parallel (unfortunately...)